lib/include/jxl/encode.h

```diff
@@ -63,18 +63,15 @@ typedef enum {
    */
   JXL_ENC_SUCCESS = 0,
 
-  /** An error occurred, for example out of memory.
+  /** An error occurred, for example out of memory, or encoder doesn't (yet)
+   * support this.
    */
   JXL_ENC_ERROR = 1,
 
   /** The encoder needs more output buffer to continue encoding.
    */
   JXL_ENC_NEED_MORE_OUTPUT = 2,
 
-  /** The encoder doesn't (yet) support this.
-   */
-  JXL_ENC_NOT_SUPPORTED = 3,
```

Comment:
I think this should rather be deprecated...

Reply:
I'd rather remove it, since it has 0 usages in our code (which could be a good indication of the amount of usages outside), and the encoder has not yet been released like the decoder

Reply:
After adding the change to the changelog I reconsidered, and added it back as deprecated.

At some point the deprecated stuff should be actually removed though

Reply:
I would suggest to do a 1.0-rc1 'release' (not really a release but a release candidate, so no commitment to maintain it) in which the deprecated stuff still exists, then a 1.0-rc2 in which it is removed, and in the actual 1.0 I would try to have a clean API.